From c6bb4b99af39daa3221c3bdc0686987ae0f31693 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Wed, 31 Jan 2018 16:47:44 +0200
Subject: [PATCH] LINGUAS: drop the languages for which upstream does not
 supply .po files

Regenerating them proved to be too painful.
Upstream has been notified: https://github.com/p11-glue/p11-kit/issues/127

Upstream-Status: Inappropriate [missing upstream distribution files]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>

---
 po/LINGUAS | 2 --
 1 file changed, 2 deletions(-)

diff --git a/po/LINGUAS b/po/LINGUAS
index 767a806c2e20..6ab48001c409 100644
--- a/po/LINGUAS
+++ b/po/LINGUAS
@@ -12,8 +12,6 @@ cy
 da
 de
 el
-en@boldquot
-en@quot
 en_GB
 eo
 es
